extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -101,22 +101,32 @@ function render_player( $track_list, $attributes ) {
 	ob_start();
 	?>
 	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
-		<ol>
-			<?php
-			foreach ( $track_list as $attachment ) :
-				printf( '<li><a href="%1$s" data-jetpack-podcast-audio="%2$s">%3$s</a></li>', esc_url( $attachment['link'] ), esc_url( $attachment['src'] ), esc_html( $attachment['title'] ) );
-			endforeach;
-			?>
+		<ol class="podcast-player__episodes">
+			<?php foreach ( $track_list as $attachment ) : ?>
+			<li class="podcast-player__episode">
+				<a
+					class="podcast-player__episode-link"
+					href="<?php echo esc_url( $attachment['link'] ); ?>"
+					data-jetpack-podcast-audio="<?php echo esc_url( $attachment['src'] ); ?>"
+					role="button"
+					aria-pressed="false"
+				>
+					<span class="podcast-player__episode-status-icon"></span>
+					<span class="podcast-player__episode-title"><?php echo esc_html( $attachment['title'] ); ?></span>
+					<time class="podcast-player__episode-duration"><?php echo esc_html( $attachment['duration'] ); ?></time>
```

Comment:
I'm getting warnings for the `duration` index:

```
[18-Mar-2020 16:14:54 UTC] PHP Notice:  Undefined index: duration in /var/www/html/wp-content/plugins/jetpack/extensions/blocks/podcast-player/podcast-player.php on line 132
[18-Mar-2020 16:14:54 UTC] PHP Stack trace:
```

I think we should check them if they are defined or not.

Reply:
I added an ` ! empty( $attachment['duration']` check on this.